src/main/java/mate/academy/springbootwebgreqit/model/Book.java

```diff
@@ -17,17 +19,24 @@ public class Book {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    @Column(nullable = false, length = 100)
+    @Column(length = 100)
```

Comment:
Left only 
```
@Column(nullable = false, length = 100)
```

same for others

Reply:
I wanted to use validation with @NotNull there, but ok)